src/applications/vaos/services/mocks/index.js

```diff
@@ -728,4 +728,4 @@ const responses = {
   // End of required v0 APIs
 };
 
-module.exports = delay(responses, 1000);
+module.exports = delay(responses, 100);
```

Comment:
I've shortened the delay from 1s to 0.1s so local testing is faster. We can increase this delay when working with loading indicators but for all other cases, a shorter delay is better for testing locally.